packages/babel-preset-babili/__tests__/preset-tests.js

```diff
@@ -0,0 +1,42 @@
+jest.autoMockOff();
+
+const babel = require("babel-core");
+const unpad = require("../../../utils/unpad");
+
+function transform(code, options = {}, sourceType = "script") {
```

Comment:
Maybe it would make sense to move `transform` to utils and share among tests to avoid duplication?